packages/material-ui-unstyled/src/SliderUnstyled/SliderUnstyled.test.js

```diff
@@ -43,6 +43,7 @@ describe('<SliderUnstyled />', () => {
     const elementRef = React.createRef();
     render(
       <SliderUnstyled
+        value={0}
```

Comment:
A bit worried about this change.  Why is it necessary? Seems like breaking change if it is required now.

Reply:
Ah this was initially needed because we were getting a the warning `Warning: Received NaN for the `value` attribute. If this is expected, cast the value to a string.` nothing broke but I'm sure users wouldn't be happy. That being said it was fixed with these changes https://github.com/mui-org/material-ui/pull/23902#discussion_r539803738 want me to revert these test changes?

Reply:
That would be appreciated. Changes to test always stand out since they're indicative of breaking changes. We should only change tests if we need to.

Reply:
Makes sense, done!